src/qmflows/cp2k_utils.py

```diff
@@ -243,16 +248,16 @@ def set_prm(settings: Settings, key: Union[str, Tuple[str, ...]],
     :data:`CP2K_KEYS_ALIAS` : :class:`dict` [:class:`str`, :class:`tuple` [:class:`str`, ...]]
         A dictionary mapping ``key_path`` aliases to the actual keys.
 
-    """  # noqa
+    """  # noqa: E501
     if isinstance(value, abc.Sequence):
-        for prm_map in value:
+        for prm_map in value:  # type: Union[MappingScalar, MappingSequence]
             set_prm(settings, key, prm_map, mol)
         return
     else:
         prm_map = copy.copy(value)
 
     try:
-        prm_key = prm_map.pop('param')
+        prm_key = cast(Union[str, Sequence[str]], prm_map.pop('param'))
```

Comment:
The [``cast()``](https://docs.python.org/3/library/typing.html#typing.cast) function is used for forcefully assign a certain type to a variable; it does nothing at runtime.

Reply:
Good to know, because in C++ for example it cast one type to another